anesthetic/samples.py

```diff
@@ -396,6 +396,9 @@ def __init__(self, *args, **kwargs):
         if root is not None:
             reader = SampleReader(root)
             samples, logL, logL_birth = reader.samples()
+            logzero = kwargs.pop('logzero', -numpy.inf)
+            logL = numpy.where(logL <= logzero, -numpy.inf, logL)
+            logL_birth = numpy.where(logL_birth <= logzero, -numpy.inf, logL_birth)
```

Comment:
In #70 you're saying 

> In principle, there shouldn't be any -numpy.inf values in the logL column, since they should be excluded in prior generation 

Why shouldn't there be `-inf` in `logL` and why do I find `-1e300` then? 

Is this what you had in mind with `logzero`?

Reply:
>Is this what you had in mind with logzero?

Yes

>Why shouldn't there be -inf in logL and why do I find -1e300 then?

In theory, samples with logzero loglikelihood should be rejected at prior generation. How were these samples generated?

Reply:
> In theory, samples with logzero loglikelihood should be rejected at prior generation. How were these samples generated?

This is output from a Cobaya+PC run...

Reply:
OK, we should keep on the lookout for any issues this might potentially cause, although it doesn't seem to have reared its head yet.